src/code/z_play.c

```diff
@@ -1557,20 +1557,20 @@ s32 Play_CameraSetAtEye(PlayState* this, s16 camId, Vec3f* at, Vec3f* eye) {
 
     camera->atLERPStepScale = 0.01f;
 
-    return ret;
+    return successfullySet;
```

Comment:
potentially an unpopular opinion, but I kinda like ret for all these instead of `successfullySet`
its alot of syllables that make reading the code less fluid to me. and it gets a little weird with the functions that pack multiple values into different bits. 

to me just leaving it as `ret` lets it flow nice, and you can explain the meaning of the return with more words in a function description. but would understand if others dont agree

Reply:
Maybe `successBits` would be a bit more accurate and simple to what it's storing? `ret` has the benefit of showing that this is the return value, but doesn't explain what it's storing, but as you mentioned could be explained via param comments